src/components/Holdings/HoldingModal.tsx

```diff
@@ -184,6 +204,8 @@ const HoldingModal = (props: ModalProps) => {
             <Button primary type="submit">
               Save
             </Button>
+            <TrashCan src={trashIcon} onClick={() => deleteHolding()} alt="Icon"/>
```

Comment:
Can we show the TrashCan component only when `props.holding` exists? 😊

Reply:
Yes! For some reason I thought we always wanted to redirect to the dashboard, this makes much more sense.